apps/demo-clarity/src/styles.scss

```diff
@@ -1,20 +1,20 @@
-$clr-header-bgColor: purple;
-$clr-header-font-color: yellow;
-$clr-header-title-font-weight: 700;
+// $clr-header-bgColor: purple;
```

Comment:
Usually , we won't allow comments like this in code so it's better to keep it as a rule even in this demo app

Reply:
done